index.html

```diff
@@ -169,9 +169,9 @@ <h4>Rs. 10,800</h4>
                         <a href="" class="btn btn-outline">Wishlist</a>
                     </div>
                 </div>
-            </div>
+            </article>
 
-            <div class="card card-vertical">
+            <article class="card card-vertical">
                 <div class="card__img">
                     <img src="./assets/dress-pic.jpg" alt="img" class="img-responsive">
```

Comment:
For image container, you could use `<figure>` tag instead of  `<div>` for semantic HTML